packages/p2p/src/pages/my-ads/create-ad-summary.jsx

```diff
@@ -49,46 +50,58 @@ const CreateAdSummary = ({ offer_amount, price_rate, type }) => {
 
             if (type === buy_sell.BUY) {
                 return (
-                    <Localize
-                        i18n_default_text="You're creating an ad to buy <0>{{ target_amount }} {{ target_currency }}</0> for <0>{{ local_amount }} {{ local_currency }}</0> <1>({{ price_rate }} {{local_currency}}/{{ target_currency }})</1>"
-                        components={components}
-                        values={values}
-                    />
+                    <Text className='create-ad-summary' weight='normal' size='xs' color='less-prominent'>
```

Comment:
```suggestion
                    <Text className='create-ad-summary' size='xs' color='less-prominent'>
```

can remove the weight since it should be normal by default